.github/workflows/tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Tests
+on: [pull_request]
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'pradyunsg'
+    container:
+      # Keep in sync with snapshot_tests/Dockerfile's base image.
+      image: mcr.microsoft.com/playwright:v1.34.0-jammy
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+      - uses: actions/setup-node@v3
+      - name: Install Node.js dependencies
+        run: npm ci
+
+      - name: Build docs
+        run: pipx run nox -s docs
```

Comment:
Huh, I copied this line: 

https://github.com/pradyunsg/furo/blob/490527b2aef00b1198770c3389a1979911ee1fcb/.github/workflows/deploy-docs.yml#L12

But it's not working here, saying pipx isn't recognized. Even though `- uses: actions/setup-python@v4` is the same.